browser/sources.gni

```diff
@@ -184,6 +184,9 @@ brave_chrome_browser_deps = [
   "//brave/components/p3a:buildflags",
   "//brave/components/playlist/common/buildflags",
   "//brave/components/privacy_sandbox",
+  "//brave/components/psst/browser/content",
```

Comment:
I don't think you need content or core here. Does gn check complain about them being missing?